literature.bib

```diff
@@ -337,6 +337,38 @@ @mastersthesis{Guething23
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2024 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{BRSK:JSS24,
+  author       = {Alexander Boll and
+                  Pooja Rani and
+                  Alexander Schulthei{\ss} and
+                  Timo Kehrer},
+  title        = {{Beyond code: Is there a difference between comments in visual and
+                  textual languages?}},
+  journal      = JSS,
+  volume       = 215,
+  pages        = {112087},
```

Comment:
Does your paper really have that many pages? :sweat_smile:  Is there a `-` missing somewhere?

Reply:
This number also struck me as crazy, but upon checking it I found the same number on Elsevier's website.

Reply:
Elsevier, a pain in the a** as always.

Reply:
indeed